system-addon/common/Actions.jsm

```diff
@@ -139,6 +140,18 @@ function AlsoToMain(action, fromTarget, skipLocal) {
   });
 }
 
+/**
+ * OnlyToThisContent - Creates a message that will only be dispatched locally.
+ *
+ * @param {object} action Any redux action (required)
+ */
+function OnlyToThisContent(action) {
```

Comment:
Is this necessary? Dispatch by default doesn't go elsewhere.

Reply:
oops I didn't realize that.